server/src/db/migrations/V1.4__Add_art.sql

```diff
@@ -1,2 +1,4 @@
 insert into `art` (`name`, `locationLat`, `locationLng`, `uri`, `type`, `creatorId`)
-values ('ART', 1, 1, 'www.google.com', 1, 1);
\ No newline at end of file
+values ('ART1', 34, -118, 'www.google.com', 1, 1);
+insert into `art` (`name`, `locationLat`, `locationLng`, `uri`, `type`, `creatorId`)
+values ('AAAAAAAAAAAAAAAAAAA', 34, -118, 'www.google.com', 1, 1);
```

Comment:
Unless you have a problem with my version, you should keep the previous version of this migration when you resolve the merge conflict.

Reply:
My reasoning to truncate it was that users can see the full title in the details page that opens once you click on the card, so having it truncated might encourage them to open it. I'm also trying to deal with the edge case that the title is so long that it overflows out of the div, since I want all my cards to be of equal height. Let me know if you still think we should wrap though